test/suite/test_backup18.py

```diff
@@ -111,6 +112,7 @@ def test_backup18(self):
         config = 'incremental=(force_stop=true)'
         bkup_c = self.session.open_cursor('backup:', None, config)
         bkup_c.close()
+        simulate_crash_restart(self, ".", "RESTART")
```

Comment:
It looks like `simulate_crash_restart` returns with the connection and session pointing to the new database. I think this is right and what I had in mind with one small addition. I think we want to create scenarios so that it tests the old code path without the restart as well as tests the crash/restart path.

Reply:
Thank you! I have added both scenarios now.